SimPEG/EM/Analytics/DC.py

```diff
@@ -0,0 +1,119 @@
+import numpy as np
+from scipy.constants import mu_0, pi
+from scipy import special
+
+def DCAnalyticHalf(txloc, rxlocs, sigma, flag="wholespace"):
+    """
+        Analytic solution for electric potential from a postive pole
+
+        Input variables:
+
+            txloc =  a xyz location of A (+) electrode (np.r_[xa, ya, za])
+
+            rxlocs = [M, N]
+                M: xyz locations of M (+) electrode (np.c_[xmlocs, ymlocs, zmlocs])
+                N: xyz locations of N (-) electrode (np.c_[xnlocs, ynlocs, znlocs])
+
+            sigma = conductivity (either float or complex)
+            flag = "wholsespace" or "halfspace"
+
+    """
+    M = rxlocs[0]
+    N = rxlocs[1]
+
+    rM = np.sqrt( (M[:,0]-txloc[0])**2 + (M[:,1]-txloc[1])**2 + (M[:,2]-txloc[1])**2 )
+    rN = np.sqrt( (N[:,0]-txloc[0])**2 + (N[:,1]-txloc[1])**2 + (N[:,2]-txloc[1])**2 )
+
+    phiM = 1./(4*np.pi*rM*sigma)
+    phiN = 1./(4*np.pi*rN*sigma)
+    phi = phiM - phiN
+
+    if flag == "halfspace":
+        phi *= 2
+
+    return phi
+
+deg2rad  = lambda deg: deg/180.*np.pi
+rad2deg  = lambda rad: rad*180./np.pi
+
+def DCAnalyticSphere(txloc, rxloc, xc, radius, sigma, sigma1, \
+                 flag = "sec", order=12, halfspace=False):
+# def DCSpherePointCurrent(txloc, rxloc, xc, radius, rho, rho1, \
+#                  flag = "sec", order=12):
+    """
+
+        Parameters:
+
+            txloc (array) : current electrode location (x,y,z)
```

Comment:
These docs won't render well with sphinx. Have a look at the mesh class docs. 

```
:param TYPE NAME: DESCRIPTION
```
ie. 
```
:param float xc: x center of depressed sphere
```